File: UNIDAD 1/EXAMEN_EU3_GR10/VIAJECITOS_REST_DOTNET_GR10/02. CLIWEB/VIAJECITOS_REST_CLIWEB_GR10/VIAJECITOS_REST_CLIWEB_GR10/Models/DetalleFactura.cs
namespace VIAJECITOS_REST_CLIWEB_GR10.Models
{
    public class DetalleFactura
    {
        public int IdDetalleFactura { get; set; }
        public int IdFactura { get; set; }
        public int IdVuelo { get; set; }
        public int Cantidad { get; set; }
        public decimal ValorUnitario { get; set; }
        public decimal Total { get; set; }
        public Vuelo Vuelo { get; set; }
    }
}

File: UNIDAD 1/00_SOA_SIN_BDD/002. REST_JAVA_CONUNI_GR10/02. CLIMOV/app/src/main/java/ec/edu/monster/modelo/DetalleFactura.kt
package ec.edu.monster.modelo

data class DetalleFactura(
    val idDetalleFactura: Int,
    val idFactura: Int,
    val idVuelo: Int,
    val cantidad: Int,
    val valorUnitario: Double,
    val total: Double,
    val vuelo: Vuelo
)